x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/total_count_header.tsx

```diff
@@ -5,24 +5,41 @@
  * 2.0.
  */
 
-import { EuiFlexItem, EuiText } from '@elastic/eui';
+import { EuiFlexItem, EuiText, EuiLoadingSpinner } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import React from 'react';
 
-export const TotalCountHeader = ({ totalCount }: { totalCount: number }) => {
+const SIGFIGS_IF_ROUNDING = 3; // Number of sigfigs to use for values < 10
+
+export const TotalCountHeader = ({
+  totalCount,
+  approximate,
+  loading,
+}: {
+  totalCount: number;
+  loading?: boolean;
+  approximate?: boolean;
+}) => {
   return (
-    <EuiFlexItem>
-      <EuiText size="s" data-test-subj="dataVisualizerTotalDocCountHeader">
+    <EuiFlexItem grow={false} style={{ flexDirection: 'row' }}>
+      <EuiText size="s" data-test-subj="dataVisualizerTotalDocCountHeader" textAlign="center">
```

Comment:
How about adding an info icon and tooltip here when sampling is being, placed between the count and the gear button, to say that random sampling is being and that this is an approximate document count?

<img width="212" alt="image" src="https://user-images.githubusercontent.com/7405507/180265739-4b386c7e-875c-4fa1-ae4d-a8c3057a10dc.png">